qcodes_contrib_drivers/drivers/Rigol/Rigol_DSG3136B.py

```diff
@@ -0,0 +1,327 @@
+"""Driver for Rigol DSG3136B microwave signal generator
```

Comment:
@EdwardLaird1 thank you for contributing the driver!

you could consider adding this information as an example notebook (https://qcodes.github.io/Qcodes/examples/writing_drivers/Creating-Instrument-Drivers.html#Documentation) so that it is build into the documentation https://qcodes.github.io/Qcodes_contrib_drivers/ :)

Reply:
Good advice. I have addressed this now; see https://github.com/QCoDeS/Qcodes_contrib_drivers/pull/356